File: app/src/main/java/com/melean/taskstimetracker/ui/view_holders/EmployeeViewHolder.java
package com.melean.taskstimetracker.ui.view_holders;

import android.view.View;
import android.widget.TextView;

import com.melean.taskstimetracker.R;
import com.melean.taskstimetracker.ui.adapters.BaseRecyclerAdapter;

public class EmployeeViewHolder extends BaseViewHolder{
    private TextView employeeName;
    public EmployeeViewHolder(View view, BaseRecyclerAdapter.OnRecyclerItemClicked onRecyclerItemClicked) {
        super(view, onRecyclerItemClicked);
    }

    @Override
    public void onBindData(View view) {
        employeeName = (TextView) view.findViewById(R.id.employee_name);
    }

    public TextView getEmployeeName() {
        return employeeName;
    }
}

File: app/src/main/java/com/melean/taskstimetracker/ui/view_holders/EmployeeViewHolder.kt
package com.melean.taskstimetracker.ui.view_holders

import android.view.View
import android.widget.TextView

import com.melean.taskstimetracker.R
import com.melean.taskstimetracker.ui.adapters.BaseRecyclerAdapter

class EmployeeViewHolder(view: View, onRecyclerItemClicked: BaseRecyclerAdapter.OnRecyclerItemClicked) : BaseViewHolder(view, onRecyclerItemClicked) {
    var employeeName: TextView? = null
        private set

    override fun onBindData(view: View) {
        employeeName = view.findViewById(R.id.employee_name) as TextView
    }
}
